src/Raven.Server/Rachis/Leader.cs

```diff
@@ -123,7 +123,7 @@ public void StepDown()
                 {
                     var status = new NodeStatus
                     {
-                        Connected = kvp.Value.Status.Equals("Connected"),
+                        Connected = kvp.Value.Status.StartsWith("Connected"),
```

Comment:
Not really happy with this. It depends on us remember that we are doing string comparisons on the values.
Better to have an explicit state field that we'll check